python/tvm/relax/analysis/analysis.py

```diff
@@ -21,14 +21,99 @@
 configuring the passes and scripting them in Python.
 """
 
-from typing import Dict, List
+from typing import Dict, List, Optional
 
 import tvm
 from tvm import tir
+from tvm.relax.ty import Type
+from tvm.relax.struct_info import StructInfo
 from tvm.relax.expr import DataflowBlock, GlobalVar, Var, Expr, Function, Binding
 from . import _ffi_api
 
 
+def get_static_type(sinfo: StructInfo) -> Type:
+    """Get the corresponding static type from a StructInfo.
+
+    Parameters
+    ----------
+    sinfo : StructInfo
+        The input struct info.
+
+    Returns
+    -------
+    ret : Type
+        The corresponding static type.
+    """
+    return _ffi_api.GetStaticType(sinfo)  # type: ignore
+
+
+def get_legacy_shape_hint(sinfo: StructInfo) -> Optional[Expr]:
+    """Get the corresponding shape from a StructInfo.
+
+    Parameters
+    ----------
+    sinfo : StructInfo
+        The input struct info.
+
+    Returns
+    -------
+    ret : Type
+        The corresponding shape.
+    """
+    return _ffi_api.GetLegacyShapeHint(sinfo)  # type: ignore
+
+
+def erase_to_well_defined(
+    sinfo: StructInfo,
+    shape_var_map: Dict[tir.Var, tir.PrimExpr] = None,
+    var_map: Dict[Var, Expr] = None,
+) -> StructInfo:
+    """Erase sinfo into a well defined form.
+
+    This function removes the StructInfo's dependencies on shape and vars that
+    are not being defined.
+
+    Parameters
+    ----------
+    sinfo : StructInfo
+        The input struct info.
+
+    shape_var_map : Dict[tir.Var, tir.PrimExpr]
+        List of shape vars defined in the scope.
+
+    var_map : Dict[Var, Expr]
+        List of vars defined in the scope.
+
+    Returns
+    -------
+    ret : StructInfo
+        The corresponding erased struct info.
+    """
+    shape_var_map = {} if shape_var_map is None else shape_var_map
+    var_map = {} if var_map is None else var_map
+
+    return _ffi_api.EraseToWellDefined(sinfo, shape_var_map, var_map)  # type: ignore
+
+
+def struct_info_lca(lhs: StructInfo, rhs: StructInfo) -> StructInfo:
+    """Unify the two struct info their least common accentor.
```

Comment:
I guess you wanted to type “ancestor”.

But it turns out that “accentor” is indeed an existing word - it’s a genus of birds 😯 https://en.wikipedia.org/wiki/Accentor.

Also please consider my feedback on looking for a better name :pray:. 
```suggestion
    """Unify the two struct info their least common ancestor.
```